.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = git://github.com/roboptim/roboptim-shared-tests.git
 [submodule "cmake"]
 	path = cmake
-	url = git://github.com/jrl-umi3218/jrl-cmakemodules.git
+	url = https://github.com/jrl-umi3218/jrl-cmakemodules
```

Comment:
Do we need this change?

Reply:
Thanks @saikishor for the feedback. For some reason, the git link on my side is timing out where as the https link allows a fast submodule pull. For this reason the following command:
```
git clone --recurse-submodules http://github.com/roboptim/roboptim-core-plugin-ipopt.git
```
is not working due to a long timeout.
The travis submodule for me is also a problem. I had to individually init and update the cmake module. 
* If you confirm that you have the same problem then I could remove also travis which is deprecated.
* If you do not have the problem, then I wonder what is the problem on my side.

Reply:
I'll check and get back

Reply:
Yes, we need this change, ref https://github.blog/security/application-security/improving-git-protocol-security-github/#no-more-unauthenticated-git